x-pack/docs/en/security/authentication/configuring-saml-realm.asciidoc

```diff
@@ -201,6 +201,14 @@ separate keys. For more information, see
 {stack-ov}/saml-guide-authentication.html#saml-enc-sign[Encryption and signing]. 
 --
 
+. (Optional) Generate service provider metadata.
++
+--
+Some identity providers can import metadata about service providers.  
+//TBD: What steps (if any) does this enable you to skip?
+You can generate SP metadata for the {stack} by using the <<saml-metadata,`elasticsearch-saml-metadata` command>>.
+--
```

Comment:
This allows you to skip some steps on the _Identity Provider_ side, none of which we document.
As well as the steps we document here for configuring the Elastic Stack, there will be some steps on the IdP side to configure ES/Kibana as a Service Provider.

For a few of them, that's as simple as importing the sp-metadata file.
For others it's a bunch of clicking around in confusing GUIs.